frontend/src/navbar/navbar.js

```diff
@@ -14,6 +14,9 @@ module.exports = app => app.component('navbar', {
   inject: ['state'],
   data: () => ({ showFlyout: false }),
   computed: {
+    dashboardView() {
+      return this.$route.fullPath.includes('dashboard');
```

Comment:
Better to make this a `startsWith` in case someone has a document with `_id === 'dashboard'`. I'd prefer if this was an explicit list based on the current route's `name` though